### PR TITLE
DOCSP-44816 mongosh 2.3.3. RN

### PR DESCRIPTION
**Pull request by kennethdyer**
## DESCRIPTION

Release notes for mongosh 2.3.3

## STAGING
* [RN](https://deploy-preview-363--docs-mongodb-shell.netlify.app/changelog/#v2.3.3)

## JIRA
[DOCSP-44816](https://jira.mongodb.org/browse/DOCSP-44816)

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [ ] Is this free of spelling errors?
- [ ] Is this free of grammatical errors?
- [ ] Is this free of staging / rendering issues?
- [ ] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

**Review by ltran-mdb2 — APPROVED** [review]
LGTM! Thanks @kennethdyer !

File: source/changelog.txt [hunk 1]
```diff
@@ -12,6 +12,42 @@ Release Notes
...
+- :issue:`MONGOSH-1868` - Align database and collection
+  aggregate functions 
+- :issue:`MONGOSH-1608` - ``mongosh`` should throw an error when
+  trying to drop a non-primary index 
+- :issue:`MONGOSH-1868` - Add support for running aggregate
+  database with a single stage
+- Revert workaround for ``depcheck`` for ``searchParams`` type
+- Fix ``depcheck`` with a more permissive workaround for the ``searchParams`` type
+- :issue:`MONGOSH-1867` - Fix incorrect ``db.createView.help``.
+- :issue:`MONGOSH-1697` - Add missing options parameter for
```
Comment (gagik) [inline source/changelog.txt:41]: The PR / ticket for this doesn't make it clear but this is also related to `.help` method for these functions similar to i.e. `createView`, might be worth framing it as that it was added to `db.collection.find.help` / `db.collection.findOne.help` instead

File: source/changelog.txt [hunk 2]
```diff
@@ -12,6 +12,42 @@ Release Notes
...
+v2.3.3
+------
+
+*Released October 30, 2024*
+
+New features released in this version:
+
+- :issue:`MONGOSH-1326` - Add ``shardedDataDistribution`` to
+  :method:`sh.status`.
+- Add individual evergreen test results with XUnit.
```
Comment (gagik) [inline source/changelog.txt:24]: Not too familiar with how we target release notes but this is an internal / not user facing change so may be worth removing

File: source/changelog.txt [hunk 3]
```diff
@@ -12,6 +12,42 @@ Release Notes
...
+Bug Fixes:
+
+- :issue:`MONGOSH-1868` - Align database and collection
+  aggregate functions 
+- :issue:`MONGOSH-1608` - ``mongosh`` should throw an error when
+  trying to drop a non-primary index 
+- :issue:`MONGOSH-1868` - Add support for running aggregate
+  database with a single stage
+- Revert workaround for ``depcheck`` for ``searchParams`` type
+- Fix ``depcheck`` with a more permissive workaround for the ``searchParams`` type
```
Comment (gagik) [inline source/changelog.txt:39]: (same as above) this was a temporary internal fix that should be removed

File: source/changelog.txt [hunk 4]
```diff
@@ -12,6 +12,42 @@ Release Notes
...
+
+Bug Fixes:
+
+- :issue:`MONGOSH-1868` - Align database and collection
+  aggregate functions 
+- :issue:`MONGOSH-1608` - ``mongosh`` should throw an error when
+  trying to drop a non-primary index 
+- :issue:`MONGOSH-1868` - Add support for running aggregate
+  database with a single stage
+- Revert workaround for ``depcheck`` for ``searchParams`` type
```
Comment (gagik) [inline source/changelog.txt:38]: this was a temporary internal fix that should be removed

**Review by gagik — COMMENTED** [review]
First release notes review  😄 so not too sure what might be important or not but just noting a couple things that may be worth considering

File: source/changelog.txt (hunk 1 above)
Comment (kennethdyer) [reply source/changelog.txt]: Fixed

File: source/changelog.txt (hunk 2 above)
Comment (kennethdyer) [reply source/changelog.txt]: Fixed

File: source/changelog.txt (hunk 3 above)
Comment (kennethdyer) [reply source/changelog.txt]: Fixed

File: source/changelog.txt (hunk 4 above)
Comment (kennethdyer) [reply source/changelog.txt]: Fixed

**Comment by kennethdyer** [comment]
@gagik, Changes applied.  For reference we start with the GitHub release notes and then let you all shave it down to the parts you want to highlight.

Anything else you'd like to see?

**Review by gagik — APPROVED** [review]
Looks good to me!

**Pull request merged**